api/lib/infrastructure/repositories/knowledge-element-repository.js

```diff
@@ -42,7 +42,7 @@ async function _findByCampaignIdForSharedCampaignParticipationWhere(campaignPart
 
 function _getByUserIdAndLimitDateQuery({ userId, limitDate }) {
```

Comment:
```suggestion
function _getByUserIdAndCreatedBefore({ userId, limitDate }) {
```
Y at-il un intérêt à l'appeler Query ?

Reply:
Je dirai que oui. Vu que la fonction retourne un queryBuilder (il est toujours possible de chaîner des `where`, `join`, etc)